riak/README.md

```diff
@@ -65,16 +69,38 @@ The Riak check is included in the [Datadog Agent][3] package, so you don't need
               pattern: \d{4}\-\d{2}\-\d{2}
     ```
 
-3. [Restart the Agent][6].
+3. [Restart the Agent][5].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][6] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                           |
+| -------------------- | ------------------------------- |
+| `<INTEGRATION_NAME>` | `riak`                          |
+| `<INIT_CONFIG>`      | blank or `{}`                   |
+| `<INSTANCE_CONFIG>`  | `{"url":"%%host%%:8098/stats"}` |
```

Comment:
Shouldn't this be `{"url":"http://%%host%%:8098/stats"}` instead?

Reply:
```suggestion
| `<INSTANCE_CONFIG>`  | `{"url":"http://%%host%%:8098/stats"}` |
```